src/flex.skl

```diff
@@ -1609,8 +1609,7 @@ int yyFlexLexer::yy_get_next_buffer()
 	/* First move last chars to start of buffer. */
 	number_to_move = (int) (YY_G(yy_c_buf_p) - YY_G(yytext_ptr) - 1);
 
-	for ( i = 0; i < number_to_move; ++i )
-		*(dest++) = *(source++);
+	memcpy(dest,source,number_to_move);
```

Comment:
The `memcpy` call lacks a `sizeof` operator in the third argument. This gives me an impression that the patch was wrong.

Reply:
`src` and `dest` are both `char*`, which I believe is guaranteed to be a single byte, so `sizeof` would return 1.

Reply:
Would you please add `* sizeof(*source)` for readability's sake? Also, the proper type for `number_to_move` would be `size_t`, or at least the length should be ensured that it's positive. Sorry, I just wished the code here to have more quality.